easybuild/easyconfigs/p/Proteinortho/Proteinortho-6.3.4-gompi-2024a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'Proteinortho'
+version = '6.3.4'
+
+homepage = 'https://www.bioinf.uni-leipzig.de/Software/proteinortho'
+description = "Proteinortho is a tool to detect orthologous genes within different species."
+
+toolchain = {'name': 'gompi', 'version': '2024a'}
```

Comment:
```suggestion
toolchain = {'name': 'foss', 'version': '2024a'}
```

Reply:
`Proteinortho` requires `LAPACK`, and will build its own bundled version if we don't provide one